alacritty_terminal/src/term/mod.rs

```diff
@@ -1741,6 +2023,8 @@ impl<T: EventListener> ansi::Handler for Term<T> {
         let end = min(start + count, cols - 1);
         let n = (cols - end).0;
 
+        self.damage.expand_line_damage(self.cursor.point.line, start, end + n);
```

Comment:
Should be just `end`, since you're deleting to the left, right?

Reply:
Ah, yes, I missed that while `end` is maxed out at `cols - 1`, `n` maxes out at `cols`.